src/modules/Dashboard/features/Respondents/Respondents.tsx

```diff
@@ -415,7 +416,28 @@ export const Respondents = () => {
         return t('noMatchWasFound', { searchValue });
       }
 
-      return appletId ? t('noRespondentsForApplet') : t('noRespondents');
+      return appletId ? (
+        <Box
+          component="span"
+          sx={{
```

Comment:
In cases where the sx prop contains more than three properties, it would be beneficial to consider using styled components. Please check other occurrences as well.